src/dodal/devices/i24/dual_backlight.py

```diff
@@ -26,18 +48,17 @@ class DualBacklight(Device):
     in "Out" position (switched off), LED2 might still be on.
     """
 
-    OUT = "Out"
-    IN = "In"
-
-    led1 = Component(EpicsSignal, "-DI-LED-01:TOGGLE")
-    pos1 = Component(BacklightPositioner, "-MO-BL-01:")
+    def __init__(self, prefix: str, name: str = "") -> None:
+        self.led1 = epics_signal_rw(LedStatus, prefix + "-DI-LED-01:TOGGLE")
+        self.pos1 = BacklightPositioner(prefix + "-MO-BL-01", name)
 
-    led2 = Component(EpicsSignal, "-DI-LED-02:TOGGLE")
+        self.led2 = epics_signal_rw(LedStatus, prefix + "-DI-LED-02:TOGGLE")
```

Comment:
Should: It would be good if these were more readable to the scientist e.g. `backlight_state` rather than `led1`?

Reply:
Renamed but I thought it was clearer to use just `backlight` and `frontlight` which is what the scientists used to refer to them. Left LED1 and LED2 in the docstring as that's what they're named in the backlight synoptic